src/services/doppler-legacy-client.ts

```diff
@@ -98,13 +98,13 @@ export function mapHeaderDataJson(json: any) {
         text: json.alert.button.text,
         url: json.alert.button.url,
       },
-      message: json.alert.type,
+      message: json.alert.message,
       type: json.alert.type,
     },
     nav: (json.nav && json.nav.map(mapNavMainEntry)) || [],
     user: {
       avatar: json.user.avatar,
-      email: json.email,
+      email: json.user.email,
```

Comment:
What a shame!, sorry again.

Reply:
It happens sometimes :) Don't worry much.